src/pages/gutenberg/free-audio-books.vue

```diff
@@ -187,17 +214,35 @@ import csvParser from 'csv-parser';
 import { logTrackerEvent } from '~/util/EventLogger';
 
 const DISPLAY_COLUMN = ['classTitle', 'classId', 'author'];
+const DISPLAY_NUMBER = 100;
 
 export default {
   name: 'FreeAudioBooks',
   layout: 'default',
+  async asyncData({ $api }) {
+    const csvData = await $api.$get(fetchGutenbergCsv());
+    const parsedData = await parseCSV(csvData);
+    const jsonLd = generateJSONLD(parsedData);
```

Comment:
`jsonLD` can be a computed property of `csvData` instead of explicitly blocking and generating it here
However I think we should considering merging the processing `csvData` in this function and in `loadCSVFile` to become one function/data

Reply:
[♻️ Simplify generate book JSONLD](https://github.com/likecoin/liker-land/pull/1620/commits/45cb67c442d0ca66439eb2b43e5327fe35b5423f)